### PR TITLE
[FLINK-20832][docs] Deliver bootstrap css and js ourselves

### PR DESCRIPTION
**Pull request by wuchong**
## What is the purpose of the change

Delivering the required files ourselves, instead of using a CDN to fix the unstable https://maxcdn.bootstrapcdn.com in China. 

## Brief change log

- Copy the resource files into `docs/page/...`
- Update the `base.html`.

## Verifying this change

I shutdowned the network on my machine, and locally build the docs, and everything works well.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (**not applicable** / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by rmetzger** [comment]
Thanks for opening the PR. Have you checked the licenses of the font files?

**Comment by wuchong** [comment]
@rmetzger , no, I just download them from the bootstrap CDN url. 
How do we deal with the existing `docs/page/font-awesome/fonts/FontAwesome.otf`?

**Comment by rmetzger** [comment]
It's mentioned in the `/NOTICE` file.

File: docs/page/css/bootstrap.min.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/*!
+ * Bootstrap v3.3.4 (http://getbootstrap.com)
+ * Copyright 2011-2015 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
```
Comment (rmetzger) [inline docs/page/css/bootstrap.min.css:4]: I believe we should mention this dependency in the `NOTICE` file and add a copy of the license to `licenses/LICENSE.bootstrap`.

Comment (wuchong) [reply docs/page/css/bootstrap.min.css:4]: Sure.

**Comment by wuchong** [comment]
@rmetzger , I removed `anchor.min.js` because the website can be loaded even without this resource.

File: NOTICE [hunk 2]
```diff
@@ -9,6 +9,10 @@ See bundled license files for details.
 
 - jQuery v3.5.1 | (c) JS Foundation and other contributors
 	-> in "docs/page/js/jquery.min.js"
+- Bootstrap v3.3.4 (http://getbootstrap.com) Copyright 2011-2015 Twitter, Inc.
+    -> css in "docs/page/css/bootstrap.min.css"
+    -> js in "docs/page/js/bootstrap.min.js"
+    -> fonts in "docs/page/fonts"
```
Comment (rmetzger) [inline NOTICE:16]:
```suggestion
- Bootstrap v3.3.4 (http://getbootstrap.com) Copyright 2011-2015 Twitter, Inc.
    -> css in "docs/page/css/bootstrap.min.css"
    -> js in "docs/page/js/bootstrap.min.js"
    -> fonts in "docs/page/fonts"

Note: The "docs/page/fonts/glyphicons-halflings*" fonts are included in Bootstrap, and licensed under the same license as Boostrap. See https://glyphicons.com/old/license.html#old-halflings-bootstrap for more information.
```

**Comment by rmetzger** [comment]
> @rmetzger , I removed anchor.min.js because the website can be loaded even without this resource.

But it seems that we are loading anchor.min.js from a cloudflare CDN.
I would ship the cloudflare files also like bootstrap ourselves.

**Comment by wuchong** [comment]
@rmetzger  I have added back the anchorjs

**Review by rmetzger — APPROVED** [review]
Thanks a lot for fixing this!

**Pull request merged**